scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -239,6 +239,9 @@ class RedundantBraces(implicit ctx: RewriteCtx) extends RewriteSession {
       case p: Term.Function if isFunctionWithBraces(p) =>
         okToRemoveAroundFunctionBody(b, true)
 
+      case _: Term.If =>
+        settings.ifElseExpressions && shouldRemoveSingleStatBlock(b)
```

Comment:
should we apply the same exactly logic to try-finally, or do-while, or for-yield? should we have three separate flags, or one flag (`ctrlExpressions`, similar to `danglingParentheses.ctrlSite`).

or even one super-generic list flag like align tokens which allows configuring matching types?:
```
rewriteTrees {
  include = [Term.If, Case]
  exclude = ...
}
```

Reply:
Up to this point, I've only seen questions about `if-else` statements. Other expressions formatting satisfies users. So I propose to limit us to just `if-else` flag until we get another feature-requests.